zaklady-js/funkce-obory/lesson.md

```diff
@@ -61,7 +61,7 @@ Hodnotu `undefined` potkáme v mnoha situacích, ale nejčastěji ve chvíli, kd
 
 ```jscon
 > const name = 'martin'
-> name.length
+> name.lenght
```

Comment:
Imho by bylo lepší tady použít nějaký okatější překlep, ať se nikomu nezaryje do paměti ta špatná varianta. Co myslíš?

Co třeba `lenkth`? :D

Reply:
A pak uvidime holky psat `name.lenkth` ;-D
Ale jsem klidne pro. Mam to updatnout? :)

Reply:
Není to nejlepší řešení. To mě jen v rychlosti napadlo, když už do toho saháš. Ale teď už je mergnuto. :D Tak to asi nech být. :) 

Na Slacku je neveřejná větší diskuze: https://dawebpraha2021.slack.com/archives/G01KZ2FRN1L/p1616754728026700

Reply:
Ahh, jasne. Zkusim na to dat duraz, at si nezapamatujou spatnej tvar. ^^